doc/examples/with_widgets_overlay/lib/example_game.dart

```diff
@@ -5,7 +5,7 @@ import 'package:flame/palette.dart';
 import 'package:flutter/material.dart';
 
 class ExampleGame extends Game with TapDetector {
-  bool isPaused = false;
+  bool isPaused = true;
```

Comment:
could this be replaced with some sort of hasOverlay check?

Reply:
An active overlay shouldn't necessarily mean that the game should be paused though, it could be an overlay that doesn't cover the full game, right?

Reply:
I have added a method `isActive` on the overlays, so this variable is not necessary anymore.